packages/kmt/kmt.0.1/opam

```diff
@@ -0,0 +1,26 @@
+opam-version: "2.0"
+synopsis: "Framework for deriving Kleene Algebras with Tests (KAT)"
+maintainer: "Michael Greenberg <michael@greenberg.science>"
+authors: "Michael Greenberg <michael@greenberg.science>"
+license: "MIT"
+homepage: "https://github.com/mgree/kmt"
+bug-reports: "https://github.com/mgree/kmt/issues"
+depends: [
+  "ocaml" {>= "4.12"}
+  "z3" {>= "4.8"}
+  "batteries" {>= "3.5"}
+  "ANSITerminal" {>= "0.8"}
+  "fmt" {>= "0.9"}
+  "alcotest" {>= "1.4"}
+  "logs" {>= "0.7"}
+  "cmdliner" {>= "1.0.4"}
+  "ppx_deriving" {>= "5.2"}
+  "dune" {>= "2.9"}
```

Comment:
```suggestion
  "dune" {>= "3.0"}
```
The linter still fails with:
```
Your dune-project file indicates that this package requires at least dune 3.0 but your opam file only requires dune >= 2.9. Please check which requirement is the right one, and fix the other.
```

Reply:
Oops---thank you! Fooled by a local switch.